.prettierrc.json

```diff
@@ -1,4 +1,10 @@
 {
   "endOfLine": "auto",
-  "htmlWhitespaceSensitivity": "ignore"
+  "htmlWhitespaceSensitivity": "ignore",
+  "overrides": [
+    {
+      "files": "*.jelly",
+      "options": { "parser": "html", "printWidth": 250 }
```

Comment:
Set `printWidth` fairly high otherwise attributes get split onto new lines, e.g.
```
<st:compress 
  xmlns:j="jelly:core" 
  xmlns:st="jelly:stapler" 
  xmlns:d="jelly:define" 
  xmlns:l="/lib/layout" 
  xmlns:t="/lib/hudson" 
  xmlns:f="/lib/form" 
  xmlns:i="jelly:fmt">
```

I don't personally find this beneficial (at least for namespace declarations).